### DISCUSSION TITLE
Checkboxes non relevant

### DISCUSSION THREAD

**Discussion by AlecAgratina** [GitbookIO/community#464] [Feature Requests]
Hello,
Don't know where to post feedback. I like the overall gitbook ergonomy (reminds me of Notion).
Just a comment. When editing an item, in the contextual menu, checkboxes are displayed to make a selection.
<img width="311" alt="gitBook_checkBoxes" src="https://github.com/GitbookIO/community/assets/157354003/25346d62-f653-4825-ab8b-09bbe7e8d547">
Since it's a " one option only choice ", it should be either radio buttons or nothing (like when creating a new item).

**Comment by addisonschultz** [comment]
Thanks for pointing this out!
